packages/augur-sdk/src/state/Controller.ts

```diff
@@ -31,25 +31,31 @@ export class Controller {
     });
   }
 
-  private updateMarketsData = async (blockNumber: number, allLogs: ParsedLog[]) => {
+  private updateMarketsData = async (marketIds: string[]) => {
+    console.log('marketIds', marketIds.length);
+    const marketsInfo = await Markets.getMarketsInfo(this.augur, await this.db, {
+      marketIds
+    });
+
+    this.augur.events.emit(SubscriptionEventName.MarketsUpdated,  {
+      marketsInfo
+    });
+  };
+
+  private allEvents = async (blockNumber: number, allLogs: ParsedLog[]) => {
     // Grab market ids from all logs.
     // Compose applies operations from bottom to top.
     const logMarketIds = fp.compose(
       fp.compact,
       fp.uniq,
+      fp.filter(NULL_ADDRESS),
```

Comment:
I'm not sure if this will work as expected. Try this:
```suggestion
      fp.filter(item => item === NULL_ADDRESS),
```

Reply:
my filter does work.

Reply:
I can make the code more explicit. I'll do that